src/transformers/generation/continuous_batching/continuous_api.py

```diff
@@ -929,6 +952,7 @@ def add_request(
         state = RequestState(
             request_id=request_id,
             initial_tokens=list(input_ids),
+            num_children=self.num_return_sequences - 1,
```

Comment:
name is weird but ok in the context of forking etc

Reply:
yes thats what i was going for